certified-connectors/App Power Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,647 @@
+            "x-ms-summary": "Card ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "name",
+            "required": true,
+            "type": "string",
+            "in": "path",
+            "x-ms-summary": "Card name",
+            "x-ms-url-encoding": "single",
+            "description": "Name of the card providing the schema.",
+            "x-ms-dynamic-values": {
+              "operationId": "MetadataListOfCards",
+              "value-path": "name",
+              "value-title": "displayName",
+              "parameters": {
+                "cardType": "ALL"
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/GetResponseSchema/{cardName}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "No need to specify response schema",
+            "schema":{}
+          }
+        },
+        "operationId": "GetResponseSchema",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "cardName",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/MetadataListOfCards/{cardType}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "List of cards",
+            "schema":{
+              "type":"array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "displayName":{
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "operationId": "MetadataListOfCards",
+        "parameters": [
+          {
+            "name": "cardType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/MetadataFormFieldTypes": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Form field types",
+            "schema":{
+              "type":"array",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "operationId": "MetadataFormFieldTypes",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/MetadataFieldTypeSpecificSchema": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "No need",
+            "schema":{}
+          }
+        },
+        "operationId": "MetadataFieldTypeSpecificSchema",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "fieldType",
+            "in": "body",
+            "required": false,
+            "schema": {}
```

Comment:
@TerhoAntila Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
This is needed